pandas/io/parsers.py

```diff
@@ -1498,9 +1498,10 @@ def _maybe_dedup_names(self, names):
                     counts[col] = cur_count + 1
 
                     if is_potential_mi:
-                        col = col[:-1] + ('%s.%d' % (col[-1], cur_count),)
+                        col = col[:-1] + ('{col}.{cnt}'.format(
+                            col=col[-1], cnt=cur_count),)
                     else:
-                        col = '%s.%d' % (col, cur_count)
+                        col = '{col}.{cnt}'.format(col=col, cnt=cur_count)
```

Comment:
Would be nice to use variable names consistently across modules, so `count` instead of `int` and `column` instead of `col`, etc...